src/lib/mina_base/transaction_logic.ml

```diff
@@ -1266,15 +1278,6 @@ module Make (L : Ledger_intf) : S with type ledger := L.t = struct
     let open Snapp_basic in
     let open Result.Let_syntax in
     (* Check timing. *)
-    let%bind timing =
-      match balance_change.sgn with
-      | Pos when not is_new ->
-          Ok a.timing
```

Comment:
We are not doing this check anymore and therefore updating the timing field of receivers as well. Looks like that's how it was in the snarks before (and now). Just want to confirm that is the case

Reply:
Indeed, this was an inconsistency! This would have given us an unsatisfiable snark work statement, which would have been bad :sweat_smile: